antlr4ide.core/src/main/java/com/github/jknack/antlr4ide/console/Console.xtend

```diff
@@ -1,38 +0,0 @@
-package com.github.jknack.antlr4ide.console
```

Comment:
That's exactly my point in https://github.com/antlr4ide/antlr4ide/issues/162 ;-).